recipes/autoconf/all/conanfile.py

```diff
@@ -32,14 +33,13 @@ def export_sources(self):
     def layout(self):
         basic_layout(self, src_folder="src")
 
-    def requirements(self):
-        self.requires("m4/1.4.19")
-
     def package_id(self):
         self.info.clear()
 
+    def requirements(self):
+        self.requires("m4/1.4.19", run=True) # m4 is called by autom4te making it a run-time requirement
```

Comment:
I think `run=True` isn't needed anymore after the `package_type="application"` was added to the m4 recipe :D

Reply:
Hi @jcar87,
Yes, good point. I have removed the "run" trait. Thanks